### ISSUE TITLE
Improve Description of "Hide Status Bar" setting.

### ISSUE THREAD

**Issue by kevincox** [Deletescape-Media/Lawnchair#2067]
## Description

Right now there is a "Desktop > Hide Status Bar" setting which is a very confusing name to me. The functional effect that I see is that when disabled the navigation bar doesn't go away when apps enter fullscreen. (I'm using gesture navigation on a Pixel 4a if that is relevant). I was trying to figure out why the nav bar was always visible and it took me a lot of searching and reading various android bug reports before realizing that this was controlled by the launcher.

I see the following options with this setting:

- It is in the "Desktop" category, even though I do not notice any effect when on the desktop/home screen/launcher.
- As far as I have noticed it doesn't actually affect the status bar at all, only the navigation bar.
- The name doesn't imply that it affects apps, not the launcher itself.

My best proposal:

- Move into another category. "Gestures" seems like the best fit because when an app is full screen you swipe from the bottom to see the nav bar, but that is admittedly a bad place as well. Maybe we need a new category?
- Change name to "Hide Navigation Bar for Fullscreen Applications"

## Environment

* Device: Google Pixel 4a
* Android version: 11
* Launcher version: 2.0-2589
* Rom: Stock

**Comment by joeldemars** [comment]
I don't think that's because of the "Hide Status Bar" option. For me, the option hides the status bar, up at the top with notifications and icons, when I'm at the home screen. I checked YouTube and didn't notice the effect that you described with the gesture bar, and the setting seems to have the intended effect. I'm using lawnchair version 2.1-2627-ci-q-merge on Android 10 on a rooted Moto G7 with gestures, with lawnchair as the QuickSwitch recents provider.

**Comment by kevincox** [comment]
Well I did more testing and you are right. While I'm fairly sure that this is the only thing I had changed to make my issue go away it seems that re-enabling this option doesn't cause the issue to recur. I guess some complex interaction is at play.

I also did more testing and see that this option does control hiding the status bar on the homescreen. Since my issue is still gone with this option re-disabled I will leave it off for now. If my issue comes back it will be an interesting data point.

But for now it seems this issue is Can't Reproduce.
